tests/pom.xml

```diff
@@ -1453,7 +1465,8 @@
                           <dependencySet>
                             <includes>
                               <!--  use openSSL in HTTP adapter -->
-                              <include>io.netty:netty-tcnative-boringssl-static</include>
+                              <include>io.netty:netty-tcnative-boringssl-static:jar:${os.detected.classifier}</include>
```

Comment:
Not strictly needed but prevents putting _all_ platform-specific jars in the build target directory (there is no single `netty-tcnative-boringssl-static` uber jar containing all different platform-specific libraries any more).